stacks-signer/src/monitoring/mod.rs

```diff
@@ -14,168 +14,180 @@
-/// Update stacks tip height gauge
-#[allow(unused_variables)]
-pub fn update_stacks_tip_height(height: i64) {
-    #[cfg(feature = "monitoring_prom")]
-    prometheus::STACKS_TIP_HEIGHT_GAUGE.set(height);
-}
-
-/// Update the current reward cycle
-#[allow(unused_variables)]
-pub fn update_reward_cycle(reward_cycle: i64) {
-    #[cfg(feature = "monitoring_prom")]
-    prometheus::CURRENT_REWARD_CYCLE.set(reward_cycle);
-}
+/// Actions for updating metrics
+#[cfg(feature = "monitoring_prom")]
+pub mod actions {
+    use ::prometheus::HistogramTimer;
+    use blockstack_lib::chainstate::nakamoto::NakamotoBlock;
+    use slog::slog_error;
+    use stacks_common::error;
+
+    use crate::config::GlobalConfig;
+    use crate::monitoring::prometheus::*;
+
+    /// Update stacks tip height gauge
+    pub fn update_stacks_tip_height(height: i64) {
+        STACKS_TIP_HEIGHT_GAUGE.set(height);
+    }
 
-/// Increment the block validation responses counter
-#[allow(unused_variables)]
-pub fn increment_block_validation_responses(accepted: bool) {
-    #[cfg(feature = "monitoring_prom")]
-    {
-        let label_value = if accepted { "accepted" } else { "rejected" };
-        prometheus::BLOCK_VALIDATION_RESPONSES
-            .with_label_values(&[label_value])
-            .inc();
+    /// Update the current reward cycle
+    pub fn update_reward_cycle(reward_cycle: i64) {
+        CURRENT_REWARD_CYCLE.set(reward_cycle);
     }
-}
 
-/// Increment the block responses sent counter
-#[allow(unused_variables)]
-pub fn increment_block_responses_sent(accepted: bool) {
-    #[cfg(feature = "monitoring_prom")]
-    {
+    /// Increment the block validation responses counter
+    pub fn increment_block_validation_responses(accepted: bool) {
         let label_value = if accepted { "accepted" } else { "rejected" };
-        prometheus::BLOCK_RESPONSES_SENT
+        BLOCK_VALIDATION_RESPONSES
             .with_label_values(&[label_value])
             .inc();
     }
-}
-
-/// Increment the number of block proposals received
-#[allow(unused_variables)]
-pub fn increment_block_proposals_received() {
-    #[cfg(feature = "monitoring_prom")]
-    prometheus::BLOCK_PROPOSALS_RECEIVED.inc();
-}
-
-/// Update the stx balance of the signer
-#[allow(unused_variables)]
-pub fn update_signer_stx_balance(balance: i64) {
-    #[cfg(feature = "monitoring_prom")]
-    prometheus::SIGNER_STX_BALANCE.set(balance);
-}
 
-/// Update the signer nonce metric
-#[allow(unused_variables)]
-pub fn update_signer_nonce(nonce: u64) {
-    #[cfg(feature = "monitoring_prom")]
-    prometheus::SIGNER_NONCE.set(nonce as i64);
-}
+    /// Increment the block responses sent counter
+    #[allow(unused_variables)]
+    pub fn increment_block_responses_sent(accepted: bool) {
+        #[cfg(feature = "monitoring_prom")]
+        {
+            let label_value = if accepted { "accepted" } else { "rejected" };
+            BLOCK_RESPONSES_SENT.with_label_values(&[label_value]).inc();
+        }
+    }
```

Comment:
Looks like you missed this function:
```suggestion
    pub fn increment_block_responses_sent(accepted: bool) {
        let label_value = if accepted { "accepted" } else { "rejected" };
        BLOCK_RESPONSES_SENT.with_label_values(&[label_value]).inc();
    }
```

Reply:
Woops, good catch!